parallel-install/pkg/preinstaller/resourcemanager_test.go

```diff
@@ -0,0 +1,101 @@
+package preinstaller
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic/fake"
+	"regexp"
+	"testing"
+)
+
+func TestResourceManager_CreateResource(t *testing.T) {
+
+	scheme := runtime.NewScheme()
+	dynamicClient := fake.NewSimpleDynamicClient(scheme)
+	cfg := getTestingConfig()
+	retryOptions := getTestingRetryOptions(cfg)
+
+	t.Run("should create resource", func(t *testing.T) {
+		// given
+		manager := NewDefaultResourceManager(dynamicClient, retryOptions)
+		resource := unstructured.Unstructured{}
+		resourceSchema := schema.GroupVersionResource{}
+
+		// when
+		err := manager.CreateResource(&resource, resourceSchema)
```

Comment:
What exactly happens here? The resource is empty and has an empty schema... I am not an expert on dynamic client but I would expect some test data here like a `namespace` for example.

Reply:
I have introduced an exemplary resource.